resources/views/manager/news/show.blade.php

```diff
@@ -46,24 +44,22 @@
                             </a>
                           </div>
                     </div>
-                    <div class="col-md-12 news-item">
-                      <p class="news-contenido">
+                    <div class="col-md-12  news-item py-3 px-3">
+                      <p class="font-times-new-roman">
                         "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
                       </p>
-                      <p class="news-contenido">
+                      <p class="font-times-new-roman">
```

Comment:
I think if in ‘news-item’ the font family should be times news roman,  this font class name needs move to the parent div. Example:

```html
<div class=“news-item font-times-new-roman ...”>
```